### ISSUE TITLE
Error in inv gaussian?

### ISSUE THREAD

**Issue by MansMeg** [LaplacesDemonR/LaplacesDemon#31]
Hi!

I think I found a bug in the invgaussian density (dinvgaussian). Now the code is as follows:
```  
dens <- log(lambda/(2 * pi * x^3)^0.5) - ((lambda * (x - 
    mu)^2)/(2 * mu^2 * x))
```
Although it should be (see https://en.wikipedia.org/wiki/Inverse_Gaussian_distribution):
```  
dens <- log(lambda^0.5/(2 * pi * x^3)^0.5) - ((lambda * (x - 
    mu)^2)/(2 * mu^2 * x))
```
i.e the square root of lambda is missing.

**Comment by singmann** [comment]
Hmm, this does indeed look like a bug. Did you maybe check what happens if you integrate the current version (with lambda different from 1)?? Does the integral indeed deviate from 1? I am just wondering if there is maybe something we are missing. If you are sure this is a mistake, can you make a pull request?

**Comment by MansMeg** [comment]
Fix in #32

**Comment by singmann** [comment]
thanks a lot, much appreciated.
